### ISSUE TITLE
Touchpad Scrolling only gets converted to up key and down key in Code Mirror and doesn't work elsewhere

### ISSUE THREAD

**Issue by danvim** [adobe/brackets#12825]
### Description

When using a laptop with multi-touch, gesture scrolling gets converted to up arrow and down arrow.

### Steps to Reproduce

1. Use a laptop with touchpad scrolling support
2. Attempt scrolling in Code Mirror
3. Attempt scrolling in sidebar

**Expected behavior:**

The elements scrolls.

**Actual behavior:**

The cursor quickly runs up and down in Code Mirror and scrolling does nothing in sidebar

### Versions

Windows 10
Brackets Release 1.7

**Comment by ficristo** [comment]
Just in case, could you try to reproduce on CodeMirror site? https://codemirror.net

**Comment by danvim** [comment]
@ficristo The CodeMirror demos scrolls normal. Meaning the cursor stayed where is was and the bar scrolled.

**Comment by danvim** [comment]
Here's the gif for context. Please note I'm only using the scrolling gesture on my laptop. And the same effect is shown in Inspector.

![getting-started](https://cloud.githubusercontent.com/assets/5255292/19463005/1f0d17ee-9523-11e6-8dcf-99240df54b39.gif)

**Comment by ficristo** [comment]
The only thing that come to my mind, this could be an issue with Chrome 49 used through [CEF](https://bitbucket.org/chromiumembedded/cef) by Brackets.
Did you try on Chrome?

If the issue is too annoying for you, you could try [brackets-electron](https://github.com/zaggino/brackets-electron/) to see if you can still hit this issue.

**Comment by danvim** [comment]
@ficristo Yes, as I have said in my first comment, I suspect the embedded Chromium is to blame because Google Chrome is totally fine.

**Comment by ficristo** [comment]
You are right! Sorry.
Marking as `cef` to verify if an upgrade will fix.
